src/eko/beta.py

```diff
@@ -46,21 +68,94 @@ def beta_1(nf):
 
     Returns
     -------
-        beta_1 : float
-            second coefficient of the QCD beta function :math:`\\beta_1^{n_f}`
+        beta_as_3 : float
+            second coefficient of the QCD beta function :math:`\\beta_as_3^{n_f}`
+
     """
     TF = constants.TR * nf
     b_ca2 = 34.0 / 3.0 * constants.CA * constants.CA
     b_ca = -20.0 / 3.0 * constants.CA * TF
     b_cf = -4.0 * constants.CF * TF
-    beta_1 = b_ca2 + b_ca + b_cf
-    return beta_1
+    beta_as_3 = b_ca2 + b_ca + b_cf
+    return beta_as_3
+
+
+@nb.njit("f8(u1)", cache=True)
+def beta_aem_3(nf):
+    """Computes the second coefficient of the QED beta function.
+
+    Implements Eq. (3.10) of :cite:`Herzog:2017ohr`.
+
+    Parameters
+    ----------
+        nf : int
+            number of active flavors
+
+    Returns
+    -------
+        beta_aem_3 : float
+            second coefficient of the QED beta function :math:`\\beta_aem_3^{n_f}`
+
+    """
+    nu = constants.uplike_flavors(nf)
+    nd = nf - nu
+    beta_aem_3 = (
+        -4.0 * constants.NC * (nu * constants.eu2**2 + nd * constants.ed2**2)
+    )
+    return beta_aem_3
 
 
 @nb.njit("f8(u1)", cache=True)
-def beta_2(nf):
+def beta_as_2aem1(nf):
```

Comment:
Can we make the names more consistent? e.g. `beta_as2_aem1` (or all underscore?)

Reply:
There is a reason for this name choice: this is the beta function for as, thus `beta_as`, while if it were for a it would have been called `beta_a`. After the second underscore, it follows the order as a series of numbers, each prefixed with the coupling, but the first one, for which it is implicit (explicitly it would be `beta_as_as2aem1`).
You can decide to put the order on the other side of the underscore, but it is not more consistent.

Think about the two cases with the same order:
- in your proposal, they would be `beta_as2_aem1` (the beta function for as) and `beta_aem1_as2` (the beta function for the qed coupling): this looks more like a mistake, why the two things are not the same?
- with current convention instead they are: `beta_as_2aem1` and `beta_a_1as2`, and if you truncate at the second underscore, you clearly read for what the beta function is, while the second piece is spelling out the order

It looks cleaner to me (even if from the point of view of the single one your own looks more natural), but I'm not especially keen in any. Both work, it's just a matter of taste at this point.

Reply:
accepted - though the correct explicit name would be `beta_qcd_as2aem1`

Reply:
I agree